docs/self-managed/identity/user-guide/authorizations/managing-resource-authorizations.md

```diff
@@ -0,0 +1,73 @@
+---
+id: managing-resource-authorizations
+title: "Managing resource authorizations"
+sidebar_label: "Managing resource authorizations"
+description: "Learn about the methods to control resource access within the Identity application."
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+In this guide you will learn about the methods to control resource access within the Identity application.
+
+### Creating resource authorizations
+
+Resource authorizations can be configured for an individual user or a group. Below we show you how to create authorizations
+for both:
+
+<Tabs groupId="entityType" defaultValue="groups" values={[{label: 'Groups', value: 'groups', }, {label: 'Users', value: 'users', },]} >
+<TabItem value="groups">
+
+1. Log in to the Identity UI and navigate to the **Groups** tab. Select the group you would like to create an authorization for from the table, and click on the **Authorizations** tab:
+
+![create-authorization-for-group-tab](../img/create-authorization-for-group-tab.png)
+
+2. Click **Create resource authorization** and a modal will open. Select the type of resource you are creating an authorization for, and click **Next**:
+
+![create-authorization-for-group-modal-1](../img/create-authorization-for-group-modal-1.png)
+
+3. Input the ID of the resource you would like to create an authorization for, select the resource from the list, and click **Next**:
+
+![create-authorization-for-group-modal-2](../img/create-authorization-for-group-modal-2.png)
+
+:::tip
+Want to apply an authorization to a wide range of resources? We support a wildcard character `*` to match any resource
```

Comment:
It might be worth stating that `*` can not be used as part of the id, f.e. `marketing-*` will not work.